content/blog/announcing_vmware_as_gold_sponsor.md

```diff
@@ -0,0 +1,19 @@
+Title: Announcing VMware as Gold Sponsor
+Date: 2018-09-29 20:00
+Category: 2018
+Slug: announcing-vmware-as-gold-sponsor
+
+<p class="text-center">
+	<a href="https://www.vmware.com/" target="_blank">
+		<img src="https://www.vmware.com/content/dam/digitalmarketing/vmware/en/files/images/wmrc/VMware_logo_gry_RGB_300dpi.jpg" alt="VMware" height="200"/>
+	</a>
+</p>
+
+Hi Everyone, it is our pleasure to announce VMware, as another PyCon India 2018’s gold sponsor.
+<!-- PELICAN_END_SUMMARY -->
+
+[VMware](https://www.vmware.com/) is a global leader in cloud infrastructure and business mobility. VMware accelerates customers’ digital transformation journey by enabling enterprises to master a software-defined approach to business and IT. With VMware solutions, organizations are building extraordinary experiences by mobilizing everything. Our customers are responding faster to opportunities with modern data and apps hosted across hybrid clouds, and safeguarding customer trust with a defense-in-depth approach to cybersecurity.
```

Comment:
please update the link to https://www.vmware.com/in/company.html and in the image as well

Reply:
Fixed the company URL but image link will remain same.